index.html

```diff
@@ -2,54 +2,59 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
     <title>Homepage</title>
 </head>
 
-<body class="page-body">
+<body>
     <header class="page-header">
-        <a class="main-logo"><img src="img/logo.svg" alt="binabox-logo">
-            <span class="logo-text">BINABOX</span></a>
         <nav class="top-nav">
+            <a class="main-logo"><img src="images\logo\binaboxlogo36x40.svg" alt="binabox logo" width="188" height="42"></a>
             <ul class="nav-list">
                 <li class="nav-item"><a href="index.html">HOME</a>
+                    <button class="dropbtn">drop arrow</button>
                 </li>
-                <li class="nav-item"><a href="about.html">ABOUT</a>
+                <li class="nav-item"><a href="about-us.html">ABOUT</a>
+                    <button class="dropbtn">drop arrow</button>
                 </li>
                 <li class="nav-item"><a href="roadmap.html">ROADMAP</a>
+                    <button class="dropbtn">drop arrow</button>
                 </li>
                 <li class="nav-item"><a href="collection.html">COLLECTION</a>
                 </li>
                 <li class="nav-item"><a href="faqs.html">FAQS</a>
+                    <button class="dropbtn">drop arrow</button>
                 </li>
                 <li class="nav-item"><a href="pages.html">PAGES</a>
+                    <button class="dropbtn">drop arrow</button>
             </ul>
             <div class="nav-buttons">
-                <button class="toggle-button" onclick="toggle()">
-                    <img id="sun-icon" src="light-mode-icon.png" alt="Toggle Light/Dark Mode">
-                </button>
-                <button type="button" class="discord-button">
-                    <img src="#" alt="discord-logo"> DISCORD
-                </button>
-                <button type="button" class="connect-button">
-                    <img src="#" alt="connect-logo"> CONNECT
+                <button class="toggle-button" onclick="toggle()">toggle
+                    <img id="sun-icon" src="images\buttons\switch.svg" width="22.5" height="22.5" alt="">
                 </button>
+                <button type="button" class="discord-button">DISCORD</button>
+                <button type="button" class="connect-button">CONNECT</button>
             </div>
         </nav>
     </header>
     <main class="page-main">
         <section class="main-info">
-            <img src="#" alt="BINABOX CUBE" class="main-image-cube">
-            <h1 class="main-heading">WITH BINABOX EXPLORE NFT COLLECTION</h1>
-            <p class="sub-heading">We are the best way to check the rarity of NFT collection</p>
+            <img src="images\index-images\binabox-cube.webp" alt="binabox cube" class="main-image-cube" width="630" height="675">

```

Comment:
pls check sizes of this image in figma 
![Screenshot 2024-08-19 at 20 23 05](https://github.com/user-attachments/assets/e134610d-6f55-4688-981c-911457dfaa42)